core/producer_server_process.py

```diff
@@ -23,13 +23,12 @@ def send_data(client: socket.socket, data: str) -> None:
     data : str
         The data to send to the client socket
     """
-
     client.send(bytes(data, "utf-8"))
     client.close()
 
 
 def receive_data(client: socket.socket, d_type: str = "str", debug: bool = False) -> Any:
-    """A function used to abstract receiving data from client
+    """Abstract receiving data from client.
```

Comment:
Codacy found an issue: [Multi-line docstring summary should start at the second line (D213)](https://app.codacy.com/manual/excalibur.krv/m3u8-dl/pullRequest?prid=6274429)